gaurun/log.go

```diff
@@ -63,10 +67,11 @@ func InitLog(outString string, levelString string) (zap.Logger, error) {
 
 	var level zap.Level
 	if err := level.UnmarshalText([]byte(levelString)); err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
-	return zap.New(encoder, level, zap.Output(writer), zap.ErrorOutput(writer)), nil
+	writeSyncer := zap.AddSync(writer)
```

Comment:
zap.Output requires zap.WriteSyncer. io.Writer needs to use zap.AddSyncer to implement zap.WriteSyncer.